PhysicsTools/NanoAOD/python/jetsAK8_cff.py

```diff
@@ -96,34 +96,35 @@
+        particleNetWithMass_ZbbvsQCD = Var("bDiscriminator('pfParticleNetDiscriminatorsJetTags:ZbbvsQCD')",float,doc="ParticleNet tagger (w/mass) Z(->bb) vs QCD discriminator",precision=10),
+        particleNetWithMass_ZccvsQCD = Var("bDiscriminator('pfParticleNetDiscriminatorsJetTags:ZccvsQCD')",float,doc="ParticleNet tagger (w/mass) Z(->cc) vs QCD discriminator",precision=10),
+        particleNet_QCD = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probQCD2hf')+bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probQCD1hf')+bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probQCD0hf')",float,doc="ParticleNet tagger QCD(0+1+2HF) sum",precision=10),
+        particleNet_QCD2HF = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probQCD2hf')",float,doc="ParticleNet tagger QCD 2 HF (b/c) score",precision=10),
+        particleNet_QCD1HF = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probQCD1hf')",float,doc="ParticleNet tagger QCD 1 HF (b/c) score",precision=10),
+        particleNet_QCD0HF = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probQCD0hf')",float,doc="ParticleNet tagger QCD 0 HF (b/c) score",precision=10),
+        particleNet_massCorr = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:masscorr')",float,doc="ParticleNet mass regression, relative correction to JEC-corrected jet mass (no softdrop)",precision=10),
+        particleNet_Xbb = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probHbb')",float,doc="ParticleNet tagger raw X->bb score. For X->bb vs QCD tagging, use Xbb/(Xbb+QCD)",precision=10),
+        particleNet_Xcc = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probHcc')",float,doc="ParticleNet tagger raw X->cc score. For X->cc vs QCD tagging, use Xcc/(Xcc+QCD)",precision=10),
+        particleNet_Xqq = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probHqq')",float,doc="ParticleNet tagger raw X->qq (uds) score. For X->qq vs QCD tagging, use Xqq/(Xqq+QCD). For W vs QCD tagging, use (Xcc+Xqq)/(Xcc+Xqq+QCD)",precision=10),
+        particleNet_Xgg = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probHgg')",float,doc="ParticleNet tagger raw X->gg score. For X->gg vs QCD tagging, use Xgg/(Xgg+QCD)",precision=10),
+        particleNet_Xtt = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probHtt')",float,doc="ParticleNet tagger raw X->tau_h tau_h score. For X->tau_h tau_h vs QCD tagging, use Xtt/(Xtt+QCD)",precision=10),
+        particleNet_Xtm = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probHtm')",float,doc="ParticleNet tagger raw X-> mu tau_h score. For X->mu tau_h vs QCD tagging, use Xtm/(Xtm+QCD)",precision=10),
+        particleNet_Xte = Var("bDiscriminator('pfParticleNetFromMiniAODAK8JetTags:probHte')",float,doc="ParticleNet tagger raw X-> e tau_h score. For X->e tau_h vs QCD tagging, use Xte/(Xte+QCD)",precision=10),
+        # explicitly combine tagger vs. QCD scores, for convenience
```

Comment:
Since you already provide all the raw scores, I don't think it's necessary to provide the combinations...? Or, if the raw scores are less likely to be used, you could provide only the combined discriminators and the raw QCD score. So that it would be possible to retrieve the raw scores if that is ever needed...

Reply:
Hi @swertz, yes indeed the information included is redundant. The original thinking was to provide each of the individual output nodes to give the user flexibility, but it would also mean that for the primary use case of discriminating the hadronic resonances from QCD it would require a selection on the ratio of two nodes. So we added the ratio too. In the end as long as the independent information from each output node is there we're happy to implement any set of the nodes in NanoAOD. Maybe @abenecke or others have an opinion there.

Reply:
I think the idea here was to have the raw scores so we have the best flexibility, but on the other hand, have the ratios to make it easier for the user to use. But I agree having the raw scores only would be enough in general and gives the best flexibility.